guides/release/tutorial/part-1/reusable-components.md

```diff
@@ -383,18 +383,6 @@ At this point, a big part of our `<Map>` template is devoted to the `<img>` tag'
 
 From within our JavaScript class, we have access to our component's arguments using the `this.args.*` API. Using that, we can move the URL logic from the template into a new getter.
 
-<div class="cta">
-  <div class="cta-note">
-    <div class="cta-note-body">
-      <div class="cta-note-heading">Zoey says...</div>
-      <div class="cta-note-message">
-        <p><code>this.args</code> is an API provided by the Glimmer component superclass. You may come across other component superclasses, such as "classic" components in legacy codebases, that provide different APIs for accessing component arguments from JavaScript code.</p>
-      </div>
-    </div>
-    <img src="/images/mascots/zoey.png" role="presentation" alt="">
-  </div>
-</div>
-
```

Comment:
Changes to the tutorial need to be made in https://github.com/ember-learn/super-rentals-tutorial.

Can you revert this change so that we can merge the pull request?